samples/cellular/nrf_cloud_multi_service/sample.yaml

```diff
@@ -18,3 +18,19 @@ tests:
       - thingy91_nrf9160_ns
     extra_args: OVERLAY_CONFIG=overlay_coap.conf
     tags: ci_build
+  sample.cellular.nrf7002ek_wifi.nrf9160dk:
+    build_only: true
+    integration_platforms:
+      - nrf9160dk_nrf9160_ns
+    platform_allow: nrf9160dk_nrf9160_ns
+    extra_args: SHIELD=nrf7002ek OVERLAY_CONFIG=overlay-nrf7002ek-wifi-scan-only.conf
+                DTC_OVERLAY_FILE="nrf91xxdk_with_nrf7002ek.overlay;nrf9160dk_with_ext_flash.overlay"
```

Comment:
I think we should (although not mandatory) use EXTRA_CONF_FILE and EXTRA_DTC_OVERLAY_FILE also here and everywhere, including line 19.